IO_Functions/GetIndices.m

```diff
@@ -0,0 +1,127 @@
+function [DatStore,Misc] = GetIndices(DatStore,Misc)
```

Comment:
why is this a new function. Did you rename the function GetIndices_US ?

Reply:
GetIndices_US was already updated to GetIndices_OptMuscleParams. The Getindices I wrote is very similar to GetIndices_OptMuscleParams. Just added the lTs variables, and removed some warnings since they were not needed anymore.